bin/setup-policy-routes.sh

```diff
@@ -199,8 +199,21 @@ create_interface_config() {
     local ether=$3
     local metric=${4:-1024}
     local tablename=$tableid
+
+    local usedns=no
+    local usentp=no
+    local usehostname=no
+    local usedomains=no
+    local dnsdefaultroute=no
+
     if [ "$tableid" = "0" ]; then
+        # This is the "primary" interface
 	tablename=main
```

Comment:
nit: indent.

Reply:
Ugh. Thanks for noticing that. My editor config is corrupt and there were all sorts of whitespace issues with that file. I've fixed this line in a revised commit, and added a new commit on top of this PR to fix the rest of the whitespace issues by replacing tabs with spaces.